README-zh_cn.md

```diff
@@ -28,17 +28,21 @@ Air 是为 Go 应用开发设计的另外一个热重载的命令行工具。只
 
 如果你只是想配置构建命令和运行命令，您可以直接使用以下命令，而无需配置文件:
 
-`air --build.cmd "go build -o bin/api cmd/run.go" --build.bin "./bin/api"`
+```shell
+air --build.cmd "go build -o bin/api cmd/run.go" --build.bin "./bin/api"
+```
 
 对于以列表形式输入的参数，使用逗号来分隔项目:
 
-`air --build.cmd "go build -o bin/api cmd/run.go" --build.bin "./bin/api" --build.exclude_dir "templates,build"`
+```shell
+air --build.cmd "go build -o bin/api cmd/run.go" --build.bin "./bin/api" --build.exclude_dir "templates,build"
+```
 
 ## 安装
 
 ### 推荐使用 install.sh
 
-```bash
+```shell
```

Comment:
what the different betweent shell and bash

Reply:
In term of highlighting none

Shell is the reference and others are aliases


https://github.com/github-linguist/linguist/blob/e2012cd67867e6287ae5ee96c8fe0b92f9a88efa/lib/linguist/languages.yml#L6530-L6537

Reply:
@xiantang as I'm unsure you saw my reply